src/shared/components/common/subscribe-button.tsx

```diff
@@ -64,7 +65,7 @@ export function SubscribeButton({
         >
           {I18NextService.i18n.t("subscribe")}
         </button>
-        <RemoteFetchModal communityActorId={actor_id} />
+        <RemoteFetchModal show={false} communityActorId={actor_id} />
```

Comment:
Why is `show` hard-coded to false?

Reply:
Going to make it optional in `modalMixin`.